LICENSE

```diff
@@ -0,0 +1,31 @@
+Copyright (c) 2017 King's College London
```

Comment:
Should that be 2018? When did you start working on this?

Reply:
Good spot! Fixed.